packages/backend/src/api/endpoints/UserApi.ts

```diff
@@ -27,6 +27,20 @@ type UserMetadataParams = {
   unsafeMetadata?: Record<string, unknown>;
 };
 
+type PasswordHasher =
+  | 'argon2i'
+  | 'argon2id'
+  | 'bcrypt'
+  | 'md5'
+  | 'pbkdf2_sha256_django'
+  | 'pbkdf2_sha1'
+  | 'scrypt_firebase';
+
+type UserPasswordHashingParams = {
+  passwordDigest: string;
+  passwordHasher: PasswordHasher;
```

Comment:
❓ Shouldn't these two be optional?

Reply:
I have made the `UserPasswordHashingParams` optional.

So you either need to provide both `passwordDigest` & `passwordHasher` or none.